tale.md

```diff
@@ -2,7 +2,7 @@
 
 This is a glimpse into the life of the one true Lord, our very own _Sri Sri Parampujya Mahatma Pranjal Tale Ji_.
 
-## Covfefe
+## Covfefe or tea
```

Comment:
@dhruvkb I think this is incorrect. Please double check and correct this.

Reply:
We are mere followers. The Lord himself has blessed tea to the rank of one of the two promised beverages so all we can do is agree.

Reply:
I hope the Lord can handle the Covfefe followers and Tea followers simultaneously. I am glad that God agrees with the Tea and hence, now completely absorbs me without any expectation of change!

Reply:
@kartikp1995 @rheaparekh Our Lord is also considering to adopt Bournvita people into his religion.